### ISSUE TITLE
Some json fields not updated when calling PUT /rest/patients/eid/ 

### ISSUE THREAD

**Issue by bw2** [phenotips/phenotips#2173]
I'm using PUT /rest/patients/eid/ to add collaborator-provided patient data (json files exported from their phenotips UI) into our local instance. Our phenotips version is v1.2.4  and collaborators' is 1.2.5

The PUT calls return 204, but I noticed some data from the json files wasn't showing up in our phenotips UI. 
To check if fields were getting dropped, I called GET /rest/patients/eid/ on the same eid as the one passed to PUT, and then compared the json structures.  It appears the following fields are ignored (with example values):

```
prenatal_perinatal_history: {u'gestation': 38}
prenatal_perinatal_phenotype: {u'prenatal_phenotype': [], u'negative_prenatal_phenotype': [{u'id': u'HP:0004325', u'label': u'Decreased body weight'}, {u'id': u'HP:0001513', u'label': u'Obesity'}, {u'id': u'HP:0000256', u'label': u'Macrocephaly'}]}
ethnicity: {u'maternal_ethnicity': [u'Caucasian'], u'paternal_ethnicity': [u'Caucasian']}
family_history: {u'consanguinity': True}
feature: {u'consanguinity': True}
nonstandard_features: [{u'observed': u'yes', u'type': u'phenotype', u'label': u'sublingual cysts'}]
global_mode_of_inheritance: [{u'id': u'HP:0000007', u'label': u'Autosomal recessive inheritance'}]
global_age_of_onset: [{u'id': u'HP:0003577', u'label': u'Congenital onset'}]
```

**Comment by marta-** [comment]
Hi @bw2, most of these had been fixed in 1.3-milestone-2. A fix for the final two was submitted today (thanks to your report): https://phenotips.atlassian.net/browse/PT-2980 and will be included in 1.3-milestone-5. Please note that we do not recommend upgrading to a milestone, you should wait for the final 1.3 instead. We plan to release 1.3 final in early January.

**Comment by bw2** [comment]
That's great. We will update in January. Thanks!
